uvicorn/main.py

```diff
@@ -345,9 +345,22 @@ def run(self, sockets=None, shutdown_servers=True):
             # We use this when the server is run from a Gunicorn worker.
             self.servers = []
             for sock in sockets:
-                server = await loop.create_server(
-                    create_protocol, sock=sock, ssl=config.ssl
-                )
+                if os.name == 'nt':
```

Comment:
Should it be `nt`?
There's also a handy built-in lib - https://docs.python.org/3.7/library/platform.html#platform.system for that.